spec/factories/early_career_payments/eligibilities.rb

```diff
@@ -9,7 +9,9 @@
       qualification { :postgraduate_itt }
       eligible_itt_subject { :mathematics }
       teaching_subject_now { true }
-      itt_academic_year { "2018_2019" }
+      itt_academic_year { "2018/2019" }
+      postgraduate_masters_loan { true }
+      postgraduate_doctoral_loan { true }
```

Comment:
@nomasprime these should no longer exist on this factory, they are now in the claims factory as parts of the following traits:

- `:with_student_loans`
- `:with_no_student_loan`

Rebasing should fix it, but might be painful as a lot has changed in the last week

Reply:
Removed.